nixos/modules/services/misc/gitlab.nix

```diff
@@ -10,9 +10,10 @@ let
   ruby = cfg.packages.gitlab.ruby;
   bundler = pkgs.bundler;
 
-  gemHome = "${cfg.packages.gitlab.env}/${ruby.gemPath}";
+  gemHome = "${cfg.packages.gitlab.ruby-env}/${ruby.gemPath}";
```

Comment:
I just noticed it just the previous `env` renamed. Would you decide to keep it, I recommend that you add a note about it to the commit message.